src/hyperion/experiment_plans/wait_for_robot_load_then_centre.py

```diff
@@ -27,15 +34,28 @@ def create_devices(context: BlueskyContext) -> GridDetectThenXRayCentreComposite
     return device_composite_from_context(context, GridDetectThenXRayCentreComposite)
 
 
-def wait_for_robot_load_then_centre(
+def wait_for_smargon_not_disabled(smargon: Smargon, timeout=60):
+    LOGGER.info("Waiting for smargon enabled")
+    SLEEP_PER_CHECK = 0.1
+    times_to_check = int(timeout / SLEEP_PER_CHECK)
+    for _ in range(times_to_check):
+        smargon_disabled = yield from bps.rd(smargon.disabled)
```

Comment:
It might be worth adding a comment somewhere saying that this this PV goes low when the robot load has finished